app/src/main/java/com/example/coronadiagnosticapp/ui/fragments/register/RegisterFragment.kt

```diff
@@ -50,6 +51,13 @@ class RegisterFragment : ScopedFragment() {
 
     }
 
+    fun hash(text: String): String {
```

Comment:
One thing about the hash, the number of hashes should be configurable (we will want to hash X times on the phone)